contents/docs/privacy/hipaa-compliance.md

```diff
@@ -37,31 +37,28 @@ There are downsides to these two solutions:
 
 PostHog offers a third approach without either of these downsides: hosting the product analytics systems yourself.
 
-## How to set PostHog up for HIPAA compliant analytics
-
-PostHog enables you to self-host on your own infrastructure and maintain full control of the data. This means you don't need to anonymize the data, nor do you need to set up a Business Associate Agreement with PostHog because you never need to send any Protected Health Information (PHI) to us in the first place. The data stays on your systems, in its original form. 
-
-You may need to sign a BAA with your hosting provider, but major providers such as Google and AWS offer these for free.
-
-### Step 1: Choose a hosting provider
+## Does PostHog offer a BAA for PostHog Cloud?
 
-We recommend hosting PostHog on your own infrastructure. If you’re leveraging a private cloud you will need a Business Associate Agreement with your provider first. These are commonly and easily available with services such as [Amazon Web Services](https://aws.amazon.com/compliance/hipaa-compliance/), [Google Cloud Platform](https://cloud.google.com/security/compliance/hipaa), [Microsoft Azure](https://docs.microsoft.com/en-us/azure/compliance/offerings/offering-hipaa-us) and many more, often for free.
+Yes. If you're interested in a BAA for HIPAA compliance, please contact [sales@posthog.com](mailto:sales@posthog.com) with information about your requirements.  
```

Comment:
Maybe better to link to the contact sales form page? Less spam, better data collection etc.

Reply:
```suggestion
Yes. If you're interested in a BAA for HIPAA compliance, please [contact us with information about your requirements](/contact-sales).  
```